style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: #6070ff;
+  --text-color: #344563;
+}
+
+body {
+  margin: 0;
+  background-image: url("./images/bg.png");
+  background-repeat: no-repeat;
+  background-position: center 76px;
+  background-size: cover;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
- [ ] Please kindly take note of the general structure of your design when compared to the given template. I couldn't see the `left-border-bottom` radius and more so, all the background images are not showing. Please ensure to align all the elements on the page according to the given template. Our ability to reproduce perfectly a given design shows that we pay attention to details. 👍 

## Figma design
<img width="141" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227379198-c625d3b2-8781-49da-8260-f2d7812506bf.png">

## Your design
<img width="300" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227379363-122774bc-a76f-486c-a3aa-2fe56f505c01.png">